extras/basic_example/public/script.js

```diff
@@ -77,9 +77,7 @@ const startBasicExample = () => {
     video: !audioOnly,
     data: true,
     screen,
-    attributes: {},
-    videoSize: [640, 480, 640, 480],
```

Comment:
👍  This probably makes sense for the basic Example since we run into incompatibilities all the time